[#421] Protocol switch-over with 'tezos-baking' (#422)
serokell/tezos-packaging: docs/baking.md

```

</details>

> ## Protocol switch-over

Reviewer: This is kind of an odd (and not very visible) position to put this section IMO.

I'd move it to be a subsection of the `Installation` one instead, since it's mostly about updates to the package itself. What do you think?

Author: Yeah, I agree that it makes sense. Though, I've renamed the section itself

> In order to have a safe transition during the protocol activation on mainnet

Reviewer: ```suggestion
In order to have a safe transition during a new protocol activation on mainnet,
```

it's required to run two sets of daemons: for the current and for the upcoming protocol.

> `tezos-baking` package aims to provide such a setup. This package is updated sometime before

Reviewer: ```suggestion
`tezos-baking` package aims to provide such a setup. This package is updated some time before
```

the new protocol is activated (usually 1-2 weeks) to run daemons for two protocols. Once the new

> protocol is activated, `tezos-baking` package is updated to stop running daemons for the old protocol.

Reviewer: ```suggestion
protocol is activated, the `tezos-baking` package is updated  again to stop running daemons for the old protocol.
```
